WikiTemplate/Crypto/wikitemplate-cryptoDesktop.md

```diff
@@ -0,0 +1,151 @@
+
+## Onboarding
+
+- [ ] Verify loading `brave://wallet` on a clean profile shows onboarding flow
+- [ ] Verify you are able to complete the onboarding flow to Ethereum create wallet by default
+- [ ] Verify you are able to complete the backup seed phrase during onboarding itself
+- [ ] Verify you can skip the onboarding from backup wallet screen and still create a wallet
+- [ ] Verify when a Dapp interacts without creating wallet opens a new tab and shows onboarding
```

Comment:
feels like there's something missing in this sentence; `without creating wallet opens a new tab` doesn't feel right, somehow

Reply:
That's the actual behaviour